docs/release-notes/march-2024.md

```diff
@@ -0,0 +1,153 @@
+---
+slug: march-2024
+hide_title: true
+title: March 2024
+description: >-
+    Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+toc_max_heading_level: 3
+tags:
+  - Release notes
+---
+
+# Semgrep release notes for March 2024
+
+## 🔧 OSS Engine
+
+* The following versions of the OSS Engine were released in September 2023:
+  * [<i class="fas fa-external-link fa-xs"></i>1.64.0](https://github.com/semgrep/semgrep/releases/tag/v1.64.0)
+  * [<i class="fas fa-external-link fa-xs"></i>1.65.0](https://github.com/semgrep/semgrep/releases/tag/v1.65.0)
+  * [<i class="fas fa-external-link fa-xs"></i>1.66.0](https://github.com/semgrep/semgrep/releases/tag/v1.66.0)
+  * [<i class="fas fa-external-link fa-xs"></i>1.66.1](https://github.com/semgrep/semgrep/releases/tag/v1.66.1)
+  * [<i class="fas fa-external-link fa-xs"></i>1.66.2](https://github.com/semgrep/semgrep/releases/tag/v1.66.2)
+  * [<i class="fas fa-external-link fa-xs"></i>1.67.0](https://github.com/semgrep/semgrep/releases/tag/v1.67.0)
+
+## 🌐 Cloud Platform
+
+### Added
+
+- The **Add to policy** button in the **Playground** can now differentiate between custom Code and Secrets rules. When you click **Add to policy**, it detects which type of rule you have written and adds the rule to the corresponding policy board. <!-- 12739 -->
+
+### Fixed
+
+- Fixed a bug in which users couldn't claim a license if they only had one organization. <!-- 13076 -->
+- **Visual Studio Code extension:** fixed an issue where rules weren't downloaded to the user's machine, which resulted in no findings detected.
+- Minor UI and in-app copy fixes in the following:
+    - Editor
+    - Settings page
+    - Tutorial page
+    - Onboarding process <!-- 13447, 13445, 13364, 13194, 13082 -->
+- Fixed a bug in which users were sometimes unable to delete their SSO configuration. <!-- 13171 -->
+
+## 💻 Code
+
+### Added
+
+- Added support for Python's `yield` keyword, enabling the detection of taint findings from taint sources returned by `yield`.
+- Added ability for users to copy file paths displayed in Semgrep Cloud Platform's **Findings** page if they aren't links.
+- Added the ability for users to see if there's a version of a rule they're currently using that supports interfile analysis.
+- Added **Clear filters** button when no findings appear in the **Findings** page after the user has set some filters.
+- **API**: added ability to get rules metadata from the API.
+
+### Changed
+
+- Code analysis started by logged-in users running `semgrep ci` now includes cross-file (intrafile) analysis. This change affects CI jobs and CLI scans.
```

Comment:
This should say interfile right?

Reply:
No. It's intr**a**file by default, int**er**file is opt-in. The actual error is "cross-file" should be "cross-function. I'll make the suggestion.